crypto/rand/rand_unix.c

```diff
@@ -77,26 +77,16 @@ static uint64_t get_timer_bits(void);
 # endif
 #endif /* defined(OPENSSL_SYS_UNIX) || defined(__DJGPP__) */
 
-#if defined(OPENSSL_RAND_SEED_NONE)
```

Comment:
why should we remove this block?

Reply:
i think it came in through #7569 and will go away once we have that on master (hopefully soon)